exporter/splunkhecexporter/client.go

```diff
@@ -89,6 +89,13 @@ func (c *client) pushMetricsData(
 		req.Header.Set("Content-Encoding", "gzip")
 	}
 
+	if md.ResourceMetrics().Len() != 0 {
+		accessToken, found := md.ResourceMetrics().At(0).Resource().Attributes().Get(splunk.HecTokenLabel)
```

Comment:
What happens if there is more than one ResourceMetrics in `md`? Looks like we will use the token from the first one and ignore the rest (I think this can happen if we batch data from different sources).
If I remember correctly signalfxexporter solves this by using a re-batcher: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/abd11c161b5f85be047258453f3f897acb3741f4/exporter/signalfxexporter/factory.go#L135

Reply:
OK, I have applied this approach.